rarm:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint1
    pid: {p: 500.0, i: 0.01, d: 100.0}

  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint2
    pid: {p: 500.0, i: 0.01, d: 100.0}

  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint3
    pid: {p: 500.0, i: 0.01, d: 100.0}

  joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint4
    pid: {p: 500.0, i: 0.01, d: 100.0}

  joint5_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint5
    pid: {p: 500.0, i: 0.01, d: 100.0}

  gripper_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: gripper_joint
    pid: {p: 500.0, i: 0.01, d: 100.0}

  grp_jnt_l1_position_controller:
    type: effort_controllers/JointPositionController
    joint: finger_joint_l
    pid: {p: 500.0, i: 0.01, d: 100.0}

  grp_jnt_r0_position_controller:
    type: effort_controllers/JointPositionController
    joint: grp_joint_r
    pid: {p: 500.0, i: 0.01, d: 100.0}

  grp_jnt_r1_position_controller:
    type: effort_controllers/JointPositionController
    joint: finger_joint_r
    pid: {p: 500.0, i: 0.01, d: 100.0}


